app/src/page/file/view.js

```diff
@@ -754,14 +775,20 @@ class FilePage extends React.PureComponent {
                           show={DateTime.SHOW_DATE} />
                       </View>
                       <View style={filePageStyle.subscriptionRow}>
+                        {((isPlayable && !fileInfo) || (isPlayable && fileInfo && !fileInfo.download_path)) &&
+                        <Button style={[filePageStyle.actionButton, filePageStyle.saveFileButton]}
+                            theme={"light"}
```

Comment:
It would be good to make these constants, though probably out of scope for this PR.